package.json

```diff
@@ -21,7 +21,6 @@
   "dependencies": {
     "@uppy/core": "^1.4.0",
     "@uppy/tus": "^1.4.0",
-    "@uppy/xhr-upload": "^1.3.0",
```

Comment:
In the tests on line https://github.com/4teamwork/vue-drop-zone/pull/31/commits/b89fe3e7186e898f1d1fa9b5c242f688b5471de5#diff-49956b8d32e7487b4aac9ff184a1564b36e00bf3c767577effabeb0233cc2ea7R5 you pass the XHRUpload. So the dependency should still exist as a dev dependency.